src/utils/DockerUtil.js

```diff
@@ -64,7 +64,8 @@ var DockerUtil = {
         port: 2376,
         ca: fs.readFileSync(path.join(certDir, 'ca.pem')),
         cert: fs.readFileSync(path.join(certDir, 'cert.pem')),
-        key: fs.readFileSync(path.join(certDir, 'key.pem'))
+        key: fs.readFileSync(path.join(certDir, 'key.pem')),
+        headers: {'user-agent': 'docker-desktop'},
```

Comment:
docker-desktop? it should be kitematic no?

Reply:
definitely, just push the correction